jolt-inlines/sha2/src/trace_generator.rs

```diff
@@ -446,75 +450,37 @@ impl Sha256SequenceBuilder {
     }
 }
 
-#[derive(Clone, Copy)]
-enum Value {
-    Imm(u64),
-    Reg(u8),
+const VIRTUAL_REGISTER_COUNT: u64 = 32; //  see Section 6.1 of Jolt paper
+pub const fn virtual_register_index(index: u64) -> u64 {
```

Comment:
I think that this function's signature should change as now register id is `u8`.